### PR TITLE
Renaming process to scenario on FE - fixes

### PR DESCRIPTION
**Pull request by mproch**

### CODE REVIEW

**Comment by dswiecki** [comment]
Something looks wired regarding branches ? why are there changes in version.sbt ?

File: ui/client/components/QueriedStateTable.js [hunk 1]
```diff
@@ -24,17 +25,19 @@ class QueriedStateTable extends React.Component {
   }
 
   render() {
+    const {t} = useTranslation()
```
Comment (JulianWielga) [inline ui/client/components/QueriedStateTable.js:28]: no.
hooks only in functional components.  https://reactjs.org/docs/hooks-overview.html
you need hoc (withTranslation) or component (Trans) here. https://react.i18next.com/latest/withtranslation-hoc
or rewrite to FC and use hook.

File: ui/client/components/tips/Errors.js [hunk 2]
```diff
@@ -65,20 +68,20 @@ export default class Errors extends React.Component {
         <div className={"node-error-links"}>
           <NodeErrorsLinkSection
             nodeIds={_.concat(otherNodeErrorIds, _.isEmpty(propertiesErrors) ? [] : "properties")}
-            message={"Errors in: "}
+            message={i18next.t("errors.errorsIn", "Errors in: ")}
```
Comment (JulianWielga) [inline ui/client/components/tips/Errors.js:71]: better move to FC (and tsx) with hook

**Review by JulianWielga — CHANGES_REQUESTED** [review]

File: ui/client/components/QueriedStateTable.js (hunk 1 above)
Comment (mproch) [reply ui/client/components/QueriedStateTable.js]: thanks for catching, used hoc for now (this should be small change to merge into 0.4.0)

File: ui/client/components/tips/Errors.js (hunk 2 above)
Comment (mproch) [reply ui/client/components/tips/Errors.js:71]: In next PR, this one should be small to be merged into 0.4.0, and for now we don't have >1 language version anyway...

**Review by JulianWielga — APPROVED** [review]

**Pull request merged**